docs/source/conf.py

```diff
@@ -50,8 +50,8 @@
 master_doc = 'index'
 
 # General information about the project.
-project = 'Bulebule'
-copyright = 'The Bulebule contributors.'
+project = 'Theseus'
+copyright = 'The Theseus contributors.'
```

Comment:
You should `git revert 049a7e8797c5ef79c19f70406f5a3280322d620e` before merging this. :wink:

Reply:
You are right, done on rebase